hugolib/site.go

```diff
@@ -909,10 +909,10 @@ func (s *Site) initializeSiteInfo() {
 		Languages:                      languages,
 		defaultContentLanguageInSubdir: defaultContentInSubDir,
 		GoogleAnalytics:                lang.GetString("GoogleAnalytics"),
-		RSSLink:                        permalinkStr(viper.GetString("RSSUri")),
+		RSSLink:                        permalinkStr(lang.GetString("RSSUri")),
 		BuildDrafts:                    viper.GetBool("BuildDrafts"),
 		canonifyURLs:                   viper.GetBool("CanonifyURLs"),
-		preserveTaxonomyNames:          viper.GetBool("PreserveTaxonomyNames"),
+		preserveTaxonomyNames:          lang.GetBool("PreserveTaxonomyNames"),
```

Comment:
It is noted in the documentation in a vague way ... and I think it should stay this vague until it is carved in stone.